doc/source/reference/random/new-or-different.rst

```diff
@@ -9,38 +9,40 @@ NumPy 1.17.0 introduced `Generator` as an improved replacement for
 the :ref:`legacy <legacy>` `RandomState`. Here is a quick comparison of the two
 implementations.
 
-================== ==================== =============
-Feature            Older Equivalent     Notes
------------------- -------------------- -------------
-`~.Generator`      `~.RandomState`      ``Generator`` requires a stream
-                                        source, called a `BitGenerator`
-                                        A number of these are provided.
-                                        ``RandomState`` uses
-                                        the Mersenne Twister `~.MT19937` by
-                                        default, but can also be instantiated
-                                        with any BitGenerator.
------------------- -------------------- -------------
-``random``         ``random_sample``,   Access the values in a BitGenerator,
-                   ``rand``             convert them to ``float64`` in the
-                                        interval ``[0.0.,`` `` 1.0)``.
-                                        In addition to the ``size`` kwarg, now
-                                        supports ``dtype='d'`` or ``dtype='f'``,
-                                        and an ``out`` kwarg to fill a user-
-                                        supplied array.
-
-                                        Many other distributions are also
-                                        supported.
------------------- -------------------- -------------
-``integers``       ``randint``,         Use the ``endpoint`` kwarg to adjust
-                   ``random_integers``  the inclusion or exclusion of the
-                                        ``high`` interval endpoint
-================== ==================== =============
+======================= ================== =============
+Feature                 Older Equivalent   Notes
+----------------------- ------------------ -------------
+`Generator`             `RandomState`      `Generator` requires a stream
+                                           source, called a `BitGenerator`
+                                           A number of these are provided.
+                                           `RandomState` uses the Mersenne
+                                           Twister `MT19937` by default,
+                                           but can also be instantiated
+                                           with any BitGenerator.
+----------------------- ------------------ -------------
+`~.Generator.random`    `random_sample`,   Access the values in a
+                        `rand`             BitGenerator, convert them to
+                                           ``float64`` in the interval
+                                           ``[0.0.,`` `` 1.0)``. In addition
```

Comment:
I wonder if ``[0.0, 1.0)`` would work as well.

Reply:
I didn't change this part of the table. I added appropriate links for the features, and then reformatted the table to fit under 80 characters width. I can merge it in.

Reply:
It would be nice to have, but rest of the PR looks good. Thanks!

Reply:
Done: 
![new-diff](https://github.com/numpy/numpy/assets/1594818/014ad9a3-003a-4369-a097-251bad83f320)